docs/content/preview/api/ycql/dml_transaction.md

```diff
@@ -51,6 +51,9 @@ transaction_block ::= START TRANSACTION ';'
 
 - An error is raised if transactions are not enabled in any of the tables inserted, updated, or deleted.
 - Currently, an error is raised if any of the `INSERT`, `UPDATE`, or `DELETE` statements contains an `IF` clause.
+- If transactions are enabled for a table, its indexes must have them enabled as well, and vice versa.
+- There is no explicit rollback.
+- DDLs are always committed immediately, even if they are inside a transaction block.
```

Comment:
May be a bit more clarity like this?

"..even if they are inside a transaction block and the transaction is rolled back, by disconnecting the session."